packages/legacy-client/src/client/net/loadObjectsPage.ts

```diff
@@ -77,6 +62,43 @@ export async function loadObjectsPage<
   );
 }
 
+export async function loadObjectsPageOrThrows<
```

Comment:
Not sure how we feel about the `orThrows`, I used that while writing this up just for me to keep track of things, but happy to rename or just remove that suffix

Reply:
Seems fine in 1.1 since the default is nothing throws. In v2, i would go for the opposite